Zend/tests/property_hooks/gh19044.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+GH-19044: Protected properties must be scoped according to their prototype (hooks variation)
```

Comment:
Nit: We have gh19044.phpt and gh19044-1.phpt (along with other gh19044.phpt's), which is a bit confusing.

Reply:
So you'd rather have attached a number to them all? I thought first one without number and the extras get one.

Reply:
Yes, or bump all the other ones by one so we don't have one with without prefix and one with `-1`. Usually we'll have:

* gh123.phpt
* gh123-2.phpt

Or:

* gh123-1.phpt
* gh123-2.phpt

I prefer the latter.